tests/integration/adapter/build-url-mixin-test.js

```diff
@@ -222,7 +222,7 @@ test('buildURL - buildURL takes the records from findMany', function(assert) {
 
   adapter.buildURL = function(type, ids, snapshots) {
     if (Array.isArray(snapshots)) {
-      return "/posts/" + snapshots.get('firstObject').belongsTo('post', { id: true }) + '/comments/';
+      return "/posts/" + snapshots[0].belongsTo('post', { id: true }) + '/comments/';
```

Comment:
Context for this change: https://github.com/emberjs/data/pull/5054#pullrequestreview-114264010